src/rt/backtrace/dwarf.d

```diff
@@ -82,24 +82,30 @@ int traceHandlerOpApplyImpl(const void*[] callstack, scope int delegate(ref size
         char[1536] buffer = void; buffer[0] = 0;
         char[256] addressBuffer = void; addressBuffer[0] = 0;
 
-        if (locations.length > 0 && locations[i].line != -1)
+        if (locations.length > 0 && locations[i].line != -1) {
             snprintf(addressBuffer.ptr, addressBuffer.length, "%.*s:%d ", cast(int) locations[i].file.length, locations[i].file.ptr, locations[i].line);
-        else
+            addressBuffer[$-1] = '\0';
```

Comment:
Oh, what is this? Just noticed this is not needed, we are in Dland after all.

Reply:
This looks like it's needed, since you are passing the `addressBuffer.ptr` to sprintf?